src/odemis/gui/model/__init__.py

```diff
@@ -1061,6 +1099,9 @@ def __init__(self, name, stage=None, stream_classes=None, fov_hw=None):
         self.show_crosshair = model.BooleanVA(True)
         self.interpolate_content = model.BooleanVA(False)
 
+        if zpos is not None:
```

Comment:
Let's call the VA `zPos`, with the uppercase, as it's the habit (or `z_pos`, but I like the uppercase better).